### PR TITLE
Support generative GQL search

### PR DESCRIPTION
**Pull request by parkerduckworth**

### CODE REVIEW

File: src/graphql/getter.test.ts [hunk 1]
```diff
@@ -1176,3 +1176,99 @@ describe('hybrid valid searchers', () => {
...
 });
+
+describe('generative search', () => {
+  const mockClient: any = {
+    query: jest.fn(),
+  };
+
+  test('singlePrompt', () => {
+    const expectedQuery =
+      '{Get{Mammal(generate(singleResult:{prompt:"When did dogs become mans best friend?"})' +
```
Comment (etiennedi) [inline src/graphql/getter.test.ts:1187]: ❤️

**Review by etiennedi — APPROVED** [review]
LGTM

File: src/graphql/generate.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    this.singlePrompt = args.singlePrompt;
+  }
+
+  toString(): string {
+    this.validate();
+
+    let str = 'generate(';
+    const results = ['error'];
+    if (this.singlePrompt) {
+      str += `singleResult:{prompt:"${this.singlePrompt.replace(/[\n\r]+/g, '')}"}`;
```
Comment (antas-marcin) [inline src/graphql/generate.ts:27]: In other clients we have methods that safely quote text when we build the argument, do we have something similar here?

Comment (parkerduckworth) [reply src/graphql/generate.ts:27]: Luckily, like the python client, this client is unaffected by this problem

**Pull request merged**